### PR TITLE
Add voice params in rtmt for experiment

### PR DESCRIPTION
**Pull request by olivMertens**
For testing and give capabilitie to test other voice with this sample - add this params

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,4 +119,14 @@ const { startSession, addUserAudio, inputAudioBufferClear } = useRealTime({
...
\ No newline at end of file
+>Sample data: The PDF documents used in this demo contain information generated using a language model (Azure OpenAI Service). The information contained in these documents is only for demonstration purposes and does not reflect the opinions or beliefs of Microsoft. Microsoft makes no representations or warranties of any kind, express or implied, about the completeness, accuracy, reliability, suitability or availability with respect to the information contained in this document. All rights reserved to Microsoft.
+
+### Change of voice
+
+If you want to change the voice for the gpt4-o voice available change in the file backend/rtmt.py the following line 172
+```python
+                params = { "api-version": "2024-10-01-preview", "deployment": self.deployment, "voice" : "nova" }
```
Comment (pamelafox) [inline README.md:128]: What about storing this in the environment instead? This feels like an environment-level customization.

Comment (olivMertens) [reply README.md:128]: yes, seems right i want to be sure that it was relevant .

**Comment by pamelafox** [comment]
Do you know if the realtime API defaults to a particular voice, or is it chosen dynamically? The code might be easier to follow if we just provide a default value, and we could align with the API default.

**Comment by olivMertens** [comment]
> Do you know if the realtime API defaults to a particular voice, or is it chosen dynamically? The code might be easier to follow if we just provide a default value, and we could align with the API default.

if it's nos updated or setted it will be 'alloy'
https://platform.openai.com/docs/guides/realtime/session

**Comment by olivMertens** [comment]
@pamelafox  the code seems ok for you ?

**Comment by pamelafox** [comment]
Doing another review now, I'll merge your changes with main and make them work with the new deployment. Thanks for your patience!

**Comment by pamelafox** [comment]
I've merged, but I'm unable to hear a different in voice choice, even when its clearly sending along a voice of "echo". @olivMertens Does this work for you with Azure? I'm asking around to see if our voice choice is more limited.

**Comment by pamelafox** [comment]
The Azure OpenAI team says that session.update must be used. More specifically:

* voice on client-sent session.update (session property): configures the voice for the lifetime of the connection, beginning with the next response
* voice on client-sent response.create (response property): configures a higher-precedence voice for the duration of an explicitly started response
* voice on server-sent session.created (session property): always includes the default session values (won't reflect any client changes)
* voice on server-sent session.updated (session property): reflects the new state of the session after a session.update from the client, including the newly-configured voice that will take effect on audio output beginning with the next response

**Comment by olivMertens** [comment]
> I've merged, but I'm unable to hear a different in voice choice, even when its clearly sending along a voice of "echo". @olivMertens Does this work for you with Azure? I'm asking around to see if our voice choice is more limited.

I've made some test before proposing and it was different each time, i will make some tests today , so i understood we have to use session update instead @pamelafox

**Comment by olivMertens** [comment]
@pamelafox  ok i verifeid only 3 voices are available and works with the azure open ai model i updated the code could you verify it ?

**Review by pamelafox — APPROVED** [review]
I have verified these changes work locally and in production. I've asked a colleague to double-check the voice_choice setting to make sure that's best practice, but it does seem to work. I'll merge after they review.

**Comment by pamelafox** [comment]
Travis confirmed this is the correct approach, merging. Thank you @olivMertens !

**Pull request merged**